easyrest/views/moderator_controller.py

```diff
@@ -128,3 +130,148 @@ def disapprove_restaurant_controller(request):
     except:
         return wrap(success=False, error="Restaurant archivation failure")
     return wrap(success=True)
+
+
+@view_config(route_name='moderator_get_users', renderer='json', request_method='GET')
+@restrict_access(user_types=["Moderator"])
+def get_users_controller(request):
+    """
+    GET user controller to return information about users for moderator
+    Args:
+        request: current pyramid request
+    Returns:
+        Json string(not pretty) created from dictionary with format:
+            {
+                "data": data,
+                "success": success,
+                "error": error
+            }
+        Where data is list with users, presented in database.
+        Style:
+            [user
+                {
+                }
+            ]
+        If user is unauthorized and not an admin - throw 403:
+    """
+    user_status = 1
+    users =\
+        request.dbsession.query(User).filter(User.role_id == user_status).all()
+    if users:
+        keys = ("id", "is_active", "status", "name", "phone_number", "email", "birth_date")
+        data = []
+        for user in users:
+            user_data = [
+                user.id,
+                user.is_active,
+                user_status,
+                user.name,
+                user.phone_number,
+                user.email,
+                user.birth_date
+            ]
+            data.append(form_dict(user_data, keys, True))
+        wrap_data = wrap(data)
+    else:
+        wrap_data = wrap([])
+    return wrap_data
+
+
+@view_config(route_name='moderator_get_owners', renderer='json', request_method='GET')
+@restrict_access(user_types=["Moderator"])
+def get_owners_controller(request):
+    """
+    GET owner controller to return information about owners for moderator
+    Args:
+        request: current pyramid request
+    Returns:
+        Json string(not pretty) created from dictionary with format:
+            {
+                "data": data,
+                "success": success,
+                "error": error
+            }
+        Where data is list with owners, presented in database.
+        Style:
+            [owner
+                {
+                }
+            ]
+        If user is unauthorized and not an admin - throw 403:
+    """
+    user_status = 2
+    users =\
+        request.dbsession.query(User).filter(User.role_id == user_status).all()
+    if users:
+        keys = ("id", "is_active", "status", "name", "phone_number", "email", "birth_date", "restaurants")
+        data = []
+        for user in users:
+            user_data = [
+                user.id,
+                user.is_active,
+                user_status,
+                user.name,
+                user.phone_number,
+                user.email,
+                user.birth_date,
+                [restaurant.name for restaurant in user.restaurants]
+            ]
+            data.append(form_dict(user_data, keys, True))
+        wrap_data = wrap(data)
```

Comment:
You'd better introduce some function which takes a model record and list of keys, and forms a dict:
```
def form_dict(record, keys):
    record = {}
    for key in keys:
        result[key] = getattr(record, key)
        # conver dates
    return result
```

Reply:
I've use such approach to make dict with certain field aliases, e.g. in DB we have certain field names. But on frontend I send fields in other names. Due to this I created keys and order values in specific way. Can I leave this? (I will use "zip" as you suggest)

Reply:
Up to you. But is there is any sense to have different field names on backend and frontend sides?

Reply:
Yes, there are no big sense on it, but in this dict I also put values, which taken not from db query: such as "user_status" in this case.

Reply:
You may form dict based on record and then add custom fields which are not in record:
```
data = form_dict(user, ("id", "is_active", "name", "phone_number", "email", "birth_date"))
data["status"] = user_status
...
```

Reply:
ok

Reply:
good, let's guys review your PR then. Please do not wait for my approve.